uint16[] id_list
float32[] input_vel_list
float32[] vel_ramp_rate_list
